themes/gatsby-theme-blog/src/theme/typography.js

```diff
@@ -0,0 +1,4 @@
+import { toTheme } from "theme-ui-typography"
+import wordpress2016 from "typography-theme-wordpress-2016"
```

Comment:
It seems a little confusing to import the fonts in a different place from the typography theme. Would it be possible/make sense to include all of the typography stuff here instead?

Reply:
That's a good point. Since the ThemeProvider is imported into both `gatsby-ssr.js` and `gatsby-browser.js`, I think that'd work